extractor/src/main/java/org/schabi/newpipe/extractor/stream/SubtitlesStream.java

```diff
@@ -1,7 +1,10 @@
 package org.schabi.newpipe.extractor.stream;
 
+import static org.schabi.newpipe.extractor.utils.Utils.EMPTY_STRING;
+
```

Comment:
Unused import, or am I missing something?

Reply:
> > Then this PR would raise that requirement to 21, right? Do we want that?
> 
> If I understand correctly, the purpose of this PR is actually to prevent raising the requirement to 21, while being able to use `LocaleCompat.forLanguageTag()`

The idea was to eventually switch to `Locale.forLanguageTag()` if/when it becomes available via desugaring.